### ISSUE TITLE
Add more body regions

### ISSUE THREAD

**Issue by udenr** [SecUSo/privacy-friendly-pain-diary#46]
We got requests for additional body regions:
- Ellbow joints
- #40

**Comment by DrDustwolf** [comment]
That's right. Please add a more fine grained method for selecting where the pain is. Also I can't select for example the spine or middle of the body both in back or front.
